exdb/static/exdb/js/search_results.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+$(document).ready(function () {
+    $('#search-results').tablesorter({
+        theme: "bootstrap",
+        widgets: ["saveSort", "columns", "filter"],
```

Comment:
Make sure the Start Time and End Time columns are being properly sorted by date & time and not just alphabetically.

Throwing a computer-friendly datetime format (ISO-8601) into a `data` attribute and writing a custom [textExtraction](https://mottie.github.io/tablesorter/docs/example-option-text-extraction.html) function for those columns would probably be the easiest solution.